### PR TITLE
Add AD studies and more

### PR DESCRIPTION
**Pull request by Aryllen**
This diff looks like I have changed the entire csv contents, even though I have not. Any assistance on this would be helpful, and if there is a problem with how the csv was updated, then this should be documented so others do not make the same mistake. Adding @milen-sage and @karawoo as reviewers for insight.

Adds:
- missing/new AD studies
- update to old MODEL-AD studies
- new keys (flourophore, Sentrix_ID, Sentrix_Row_Column, treatmentDose)
- new platforms (illuminaHumanMethylationEPIC, Siemens Prisma)
- new libraryPreparationMethod (KAPA)
- new treatmentType (verubecestat)

### CODE REVIEW

**Comment by karawoo** [comment]
I'm guessing the diff is an issue of windows vs. unix line endings

**Comment by Aryllen** [comment]
I did this on my Linux VM so I wouldn't get weird Windows artifacts.

**Comment by karawoo** [comment]
Ah, it looks like there might have been an extra empty column or two added when you saved the file? See the commas at the end of the line on the right side here, vs. the left side which doesn't have them (or has fewer)

Screen Shot 2020-07-09 at 4 31 35 PM

**Comment by Aryllen** [comment]
Ugh. Thank you finding that. I will fix. 🤦

File: synapseAnnotations/sage_controlled_vocabulary_schema.csv [hunk 1]
```diff
@@ -542,7 +550,7 @@ Il1rapKO,"triple mutant strain carries a humanized ApoE knock-in mutation, a 387
 Kif21bT82TSNP,"triple mutant strain carrying a T82T point mutation of the Kif21b gene, a humanized ApoE knock-in mutation, and an R47H point mutation of Trem2",,,,FALSE,modelSystemName,,https://www.jax.org/strain/031938
 MAPT-P301K,,,,,FALSE,modelSystemName,,
 Meox2KOHET,"triple mutant strain carrying a humanized ApoE knock-in mutation, a Cre recombinase knock-in/knock-out allele of the Meox2 gene, and an R47H point mutation of Trem2",,,,FALSE,modelSystemName,,https://www.jax.org/strain/033770
-MthfrC677TSNP,triple mutant strain carrying a humanized ApoE knock-in mutation; an R47H point mutation of Trem2; and a mutant of the Mthfr gene carrying a point mutation that models the human C677T polymorphism,,,,FALSE,modelSystemName,,https://www.jax.org/strain/030922
+MthfrC677TSNP,triple mutant strain carrying a humanized ApoE knock-in mutation, an R47H point mutation of Trem2, and a mutant of the Mthfr gene carrying a point mutation that models the human C677T polymorphism,,,,FALSE,modelSystemName
```
Comment (Aryllen) [inline synapseAnnotations/sage_controlled_vocabulary_schema.csv:553]: This is interesting. I didn't change the semi-colons, but apparently my app did.

**Comment by Aryllen** [comment]
Okay, so this is looking better, but I did find the main issue. The app on my Linux VM changed the semi-colons to commas, which messed up the column spacing on rows with semi-colons. I will fix tomorrow.

**Comment by Aryllen** [comment]
Closing since duplicate with #626.

**Pull request closed without merging**